### ISSUE TITLE
 ns-process-data ERROR

### ISSUE THREAD

**Issue by JakubSudik123** [nerfstudio-project/nerfstudio#3112]
after this command: ns-process-data images --data {DATA_PATH} --output-dir {PROCESSED_DATA_DIR}

*** Aborted at 1714161325 (unix time) try "date -d @1714161325" if you are using GNU date ***
    @     0x7ffe613a4132 log2f
    @     0x7ff6ef5c2698 `__scrt_common_main_seh'::`1'::filt$0
    @     0x7ffe5d8fe390 __C_specific_handler
    @     0x7ffe63c9440f __chkstk
    @     0x7ffe63c0e466 RtlFindCharInUnicodeString
    @     0x7ffe63c933fe KiUserExceptionDispatcher
    @     0x7ffe34bd264e void __cdecl __ExceptionPtrRethrow(void const * __ptr64)
    @     0x7ffca083916e public: void __cdecl c10::ivalue::Future::markCompleted(void) __ptr64
    @     0x7ffca0bc9d42 struct _object * __ptr64 __cdecl THPGenerator_initDefaultGenerator(struct at::Generator)
    @     0x7ffca010dec5 (unknown)
    @     0x7ffca0666b29 PyInit_pycolmap
    @     0x7ffe3a591080 (unknown)
    @     0x7ffe3a5926a5 __NLG_Return2
    @     0x7ffe63c93c56 RtlCaptureContext2
    @     0x7ffca0110936 (unknown)
    @     0x7ffdcc616f90 cfunction_call_varargs
    @     0x7ffdcc615f56 _PyObject_MakeTpCall
    @     0x7ffdcc62269b property_descr_get
    @     0x7ffdcc65e37a _PyObject_GenericGetAttrWithDict
    @     0x7ffdcc65d969 PyObject_GetAttrString
    @     0x7ffca0145dfe PyInit_pycolmap
    @     0x7ffca019d98d PyInit_pycolmap
    @     0x7ffca017628d PyInit_pycolmap
    @     0x7ffca0133ed0 (unknown)
    @     0x7ffca01407a8 PyInit_pycolmap
    @     0x7ffca01402ae PyInit_pycolmap
    @     0x7ffdcc7221fd _PyImport_LoadDynamicModuleWithSpec
    @     0x7ffdcc7218c6 _imp_create_dynamic_impl
    @     0x7ffdcc6594e5 cfunction_vectorcall_FASTCALL
    @     0x7ffdcc61609e PyVectorcall_Call
    @     0x7ffdcc6f3a50 do_call_core
    @     0x7ffdcc6f05a1 _PyEval_EvalFrameDefault

**Comment by jb-ye** [comment]
your local colmap installation seems broken.

**Comment by PLAN8VR** [comment]
I have the same issue, treble checked the environment path n windows

**Comment by PLAN8VR** [comment]
I have made a little progress with this I believe, first I had to add `HOME` in systems environment variables as my Windows User root directory, then the `ns-install-cli` cmd seems to nearly complete, but I get two errors....

`[22:34:04] ❌ Completion script generation failed: ['ns-process-data', '--tyro-print-completion', 'bash'] install.py:123
           *** Aborted at 1714944844 (unix time) try "date -d @1714944844" if you are using GNU date ***  install.py:127
               @     0x7ffafdda4132 log2f
               @     0x7ff686792698 OPENSSL_Applink
               @     0x7ffaf888e390 __C_specific_handler
               @     0x7ffb0029440f __chkstk
               @     0x7ffb0020e466 RtlFindCharInUnicodeString
               @     0x7ffb002933fe KiUserExceptionDispatcher
               @     0x7ffa9db4264e void __cdecl __ExceptionPtrRethrow(void const * __ptr64)
               @     0x7ff93494916e public: void __cdecl c10::ivalue::Future::markCompleted(void) __ptr64
               @     0x7ff934cd9d42 struct _object * __ptr64 __cdecl
           THPGenerator_initDefaultGenerator(struct at::Generator)
               @     0x7ff912d9dec5 (unknown)
               @     0x7ff9132f6b29 PyInit_pycolmap
               @     0x7ffa53a01080 (unknown)
               @     0x7ffa53a026a5 __NLG_Return2
               @     0x7ffb00293c56 RtlCaptureContext2
               @     0x7ff912da0936 (unknown)
               @     0x7ffa37cd6f90 _PyMethodDef_RawFastCallKeywords
               @     0x7ffa37cd5f56 _PyObject_MakeTpCall
               @     0x7ffa37ce269b PyWrapper_New
               @     0x7ffa37d1e37a _PyObject_GenericGetAttrWithDict
               @     0x7ffa37d1d969 PyObject_GetAttrString
               @     0x7ff912dd5dfe PyInit_pycolmap
               @     0x7ff912e2d98d PyInit_pycolmap
               @     0x7ff912e0628d PyInit_pycolmap
               @     0x7ff912dc3ed0 (unknown)
               @     0x7ff912dd07a8 PyInit_pycolmap
               @     0x7ff912dd02ae PyInit_pycolmap
               @     0x7ffa37de21fd PyImport_AppendInittab
               @     0x7ffa37de18c6 PyImport_Import
               @     0x7ffa37d194e5 _PyCFunction_DebugMallocStats
               @     0x7ffa37cd609e PyVectorcall_Call
               @     0x7ffa37db3a50 PyEval_GetFuncDesc
               @     0x7ffa37db05a1 _PyEval_EvalFrameDefault

           ❌ Completion script generation failed: ['ns-process-data', '--tyro-print-completion', 'zsh']  install.py:123
           *** Aborted at 1714944844 (unix time) try "date -d @1714944844" if you are using GNU date ***  install.py:127
               @     0x7ffafdda4132 log2f
               @     0x7ff686792698 OPENSSL_Applink
               @     0x7ffaf888e390 __C_specific_handler
               @     0x7ffb0029440f __chkstk
               @     0x7ffb0020e466 RtlFindCharInUnicodeString
               @     0x7ffb002933fe KiUserExceptionDispatcher
               @     0x7ffa9db4264e void __cdecl __ExceptionPtrRethrow(void const * __ptr64)
               @     0x7ff93494916e public: void __cdecl c10::ivalue::Future::markCompleted(void) __ptr64
               @     0x7ff934cd9d42 struct _object * __ptr64 __cdecl
           THPGenerator_initDefaultGenerator(struct at::Generator)
               @     0x7ff912d9dec5 (unknown)
               @     0x7ff9132f6b29 PyInit_pycolmap
               @     0x7ffa53a01080 (unknown)
               @     0x7ffa53a026a5 __NLG_Return2
               @     0x7ffb00293c56 RtlCaptureContext2
               @     0x7ff912da0936 (unknown)
               @     0x7ffa37cd6f90 _PyMethodDef_RawFastCallKeywords
               @     0x7ffa37cd5f56 _PyObject_MakeTpCall
               @     0x7ffa37ce269b PyWrapper_New
               @     0x7ffa37d1e37a _PyObject_GenericGetAttrWithDict
               @     0x7ffa37d1d969 PyObject_GetAttrString
               @     0x7ff912dd5dfe PyInit_pycolmap
               @     0x7ff912e2d98d PyInit_pycolmap
               @     0x7ff912e0628d PyInit_pycolmap
               @     0x7ff912dc3ed0 (unknown)
               @     0x7ff912dd07a8 PyInit_pycolmap
               @     0x7ff912dd02ae PyInit_pycolmap
               @     0x7ffa37de21fd PyImport_AppendInittab
               @     0x7ffa37de18c6 PyImport_Import
               @     0x7ffa37d194e5 _PyCFunction_DebugMallocStats
               @     0x7ffa37cd609e PyVectorcall_Call
               @     0x7ffa37db3a50 PyEval_GetFuncDesc
               @     0x7ffa37db05a1 _PyEval_EvalFrameDefault`


It seems like I'm nearly there, just need a hand to sort out whatever's going wrong with this bit!

**Comment by jb-ye** [comment]
@PLAN8VR @JakubSudik123 Could you try to re-install pycolmap? (I believe the problem came from pycolmap not colmap). 

```pip install --force-reinstall pycolmap```

**Comment by PLAN8VR** [comment]
Hi @jb-ye , thank you for taking a look at this. In the end I fixed it by doing a total conda environment reinstall, but I think the main thing I discovered was that I had to add `$home` to the environment variables with the path to my windows user directory. 

I came to this conclusion as when I went to run the `ns-install-cli` command, it gave an error that tis was missing - I suspect that this was the issue overall.

However, it seems a really odd error to find, like, I don't really understand why that was needed in the first place?
